src/components/core/Alert.vue

```diff
@@ -56,8 +61,19 @@ export default {
   },
 
   methods: {
+    ...mapActions(['setAlert']),
     getColor (type) {
       return this.colors.get(type) || ''
+    },
+    /**
+     * Dismisses the alert from the UI, also removing it from the Vuex store.
+     *
+     * @param {Function} toggleFunction - the original Vuetify toggle function
+     * @see https://vuetifyjs.com/en/api/v-alert/
+     */
+    closeAlert (toggleFunction) {
```

Comment:
This is fixing #625 . See the `@click` in the Vuetify Alert component, which is bound to this `closeAlert` function, effectively removing the alert from the store too :+1: e2e test added for this.

(`toggleFunction` is the `toggle` function slot prop from the Vuetify docs, which was called before to remove the Alert component from the UI.... we still call that, but first we remove the alert from the Vuex store...)